pkg/platform/microservice/configFiles/repo.go

```diff
@@ -54,22 +56,28 @@ func (r k8sRepo) GetConfigFilesNamesList(applicationID string, environment strin
 		return data, err
 	}
 
-
 	for name := range configMap.BinaryData {
 		data = append(data, name)
 	}
 
 	return data, nil
 }
 
-func (r k8sRepo) AddEntryToConfigFiles(applicationID string, environment string, microserviceID string, data platform.StudioConfigFile) error {
+func (r k8sRepo) AddEntryToConfigFiles(applicationID string, environment string, microserviceID string, data MicroserviceConfigFile) error {
 
 	// Get name of microservice
 	name, err := r.k8sDolittleRepo.GetMicroserviceName(applicationID, environment, microserviceID)
 	if err != nil {
 		return errors.New("unable to find microservice")
 	}
 
+	logContext := r.logContext.WithFields(logrus.Fields{
+		"method":          "GetConfigFilesNamesList",
```

Comment:
```suggestion
		"method":          "AddEntryToConfigFiles",
```

Method should be the methods name where the logger is defined.